### ISSUE TITLE
defaulted loans amount  not really changing

### ISSUE THREAD

**Issue by krosza** [Augmint/augmint-sim#16]
cannot produce higher default amount than 12000-18000

**Comment by szerintedmi** [comment]
that's the achievement of a new monetary board actor ( https://github.com/DecentLabs/augmint-sim/commit/3f0270e33e8c95b6e186e29409830396d56d24c0)
It adjusts the loan collateral ratio based on ETH/USD trendline steepness. You can follow the collateral ratio on a new graph. 
 These are the parameters of the actor for now:
```
boardLoanCollateralRatio: {
    type: 'BoardLoanCollateralRatio',
    balances: {},
    params: {
        HIGH_COLLATERAL_RATIO: 0.3, // collateral ratio when ETH/USD trend above high trigger
        MID_COLLATERAL_RATIO: 0.6, // collateral ratio when ETH/USD trend b/w low & high triggers
        LOW_COLLATERAL_RATIO: 0.6, // collateral ratio when ETH/USD trend < low trigger
        TREND_TRIGGER_LOW: 0.003, // ETH/USD trend: least squares abs(m) value (price normalised to 0-1)
        TREND_TRIGGER_HIGH: 0.01, // ETH/USD trend: least squares abs(m) value (price normalised to 0-1)
        MIN_DAYS_BEFORE_RAISE: 14 // how many days to wait from last change before raising COLLATERAL ratio
    }
``` 
The trend is calculated using least squares method, number of days is based on this param:
`ethUsdTrendSampleDays: 5, // how many days to inspect for rates.ethToUsdTrend calculation)`

The borrower actors have been modified too.  They take the collateral ratio into account based on this param:
`COLLATERAL_RATIO_SENSITIVITY: 2, // chance = COLLATERAL_RATIO_SENSITIVITY * collateralRatio` 
Basically if the collateral ratio is higher then higher the chance that they take a loan.
